examples/mongodbatlas_stream_privatelink_endpoint/confluent_dedicated_cluster/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    mongodbatlas = {
+      source  = "mongodb/mongodbatlas"
+      version = "1.24.0"
```

Comment:
1.25? Or did we decide not to include these?

Reply:
I seem to remeber that if we put a version that does not exist yet it fails, might be wrong though

Reply:
I would change it to  `>~` to use latest 1.x by default

Reply:
We decided not to include the TF version which is the `required_version ` below

Reply:
For our provider TF version I think we can try  >~ as suggested by @EspenAlbert . I think we have a backlog ticket to fix this in the job at some point